hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java

```diff
@@ -126,75 +126,63 @@ public static long formatDateTime(String date) throws ParseException {
         .toInstant().toEpochMilli();
   }
 
-
-
   /**
    * verifies that bucket name / volume name is a valid DNS name.
    *
    * @param resName Bucket or volume Name to be validated
    *
    * @throws IllegalArgumentException
    */
-  public static void verifyResourceName(String resName)
-      throws IllegalArgumentException {
-
+  public static void verifyResourceName(String resName) throws IllegalArgumentException {
     if (resName == null) {
       throw new IllegalArgumentException("Bucket or Volume name is null");
     }
 
-    if ((resName.length() < OzoneConsts.OZONE_MIN_BUCKET_NAME_LENGTH) ||
-        (resName.length() > OzoneConsts.OZONE_MAX_BUCKET_NAME_LENGTH)) {
+    if (resName.length() < OzoneConsts.OZONE_MIN_BUCKET_NAME_LENGTH ||
+        resName.length() > OzoneConsts.OZONE_MAX_BUCKET_NAME_LENGTH) {
       throw new IllegalArgumentException(
-          "Bucket or Volume length is illegal, " +
-              "valid length is 3-63 characters");
+          "Bucket or Volume length is illegal, valid length is 3-63 characters");
     }
 
-    if ((resName.charAt(0) == '.') || (resName.charAt(0) == '-')) {
+    if (resName.charAt(0) == '.' || resName.charAt(0) == '-') {
       throw new IllegalArgumentException(
           "Bucket or Volume name cannot start with a period or dash");
     }
 
     if ((resName.charAt(resName.length() - 1) == '.') ||
```

Comment:
I see the braces are removed everywhere, to be consistent can we remove from here too?

Reply:
`if (resName.charAt(resName.length() - 1) == '-' || resName.charAt(resName.length() - 1) == '-') {`

Reply:
Done. Thanks for the review @bharatviswa504